website/docs/concepts/comparisons/index.mdx

```diff
@@ -86,3 +86,24 @@ Great tools when it comes at scheduling long-running jobs. However, they are not
 1. When you want to implement modular APIs and backends with instantaneous responses
 2. When you want to interact with workflow engines and control over the operations they perform, including pooling and retries
 3. When you look for a serverless deployment solution and want to focus on your business logic
+
+
+|  | Metatype | Development Platforms | Headless CMS | Low code platform |
+|------------------------------------|------------------------|-------------|------------------|------------------------------------|
+| Interoperability and Integrability | Moderate | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: |
+| Custom Code Support | Moderate | Moderate | :x: | :heavy_check_mark: |
+| External Module Integration through Function Calls | :x: | :x: | :x: | :heavy_check_mark: |
+| Scalability | High | High | Moderate | High |
+| Building Complex APIs | :heavy_check_mark: | :x: | :x: | :heavy_check_mark: |
+| Workflow Automation | Moderate | Moderate | High | High |
+| Collaboration Tools | Moderate | Moderate | High | Moderate |
+| Monitoring and Analytics | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: | :x: |
+| Graphical User Interface | Low | High | High | Very High |
+| Development Time | Slow | Fast | Fast | Fast |
+| Development SDK | :heavy_check_mark: | :x: | :x: | :heavy_check_mark: |
+| Development Time | Slow | Fast | Fast | Fast |
+| Ease of Deployment | Hard | Easy | Easy | Very Easy |
+| Versioning | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: | :x: |
+| Open Source | :heavy_check_mark: | :x: | :x: | :heavy_check_mark: |
+
+{/* TODO: Add Legend for the parameters */}
```

Comment:
Yeah, the mixture of emoji and text is a bit disorienting. Maybe convert the emoji to text?

Reply:
Good point. Another solution can be sorting them out, top with text and bottom with emoji. But let's go with urs.